source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -160,7 +160,17 @@ def tag_journeys(resp):
     """
     qualify the journeys
     """
-    pass
+    car = next((j for j in resp.journeys if helpers.is_car_direct_path(j)), None)
+    if car is None:
+        return
+    for j in resp.journeys:
+        if not j.HasField('co2_emission'):
```

Comment:
I'll add a test for the field in the car section. If there is no data for car, we'll do not add any ecologic tag. Thus, if you add the CO2 for car, you'll need to add it to all the non 0 other physical mode.